### ISSUE TITLE
Write-interactions silently fail using Warp Contracts

### ISSUE THREAD

**Issue by erhant** [textury/arlocal#152]
Hi, there seems to be a problem with the latest 2 versions of ArLocal (v1.1.63 and v1.1.64) that causes write-interactions using Warp Contracts (via `WarpFactory.forLocal`) to silently fail.

I have created an issue on Warp: https://github.com/warp-contracts/warp/issues/494

Creating an issue here as well if you might have an idea why this is happening.

Thanks!

**Comment by ppedziwiatr** [comment]
Hey, I've found the root cause (testing on latest ArLocal version - `1.1.64`)

This works:

<img width="1248" alt="image" src="https://github.com/textury/arlocal/assets/84311757/53a40a7a-4286-42a4-938b-6d1c44f0b5ca">

Query:
```
query {
  transactions(
    ids: ["thLz97087UrlLfXJ0RphXZ7g_mK569ro63ulCxxRtdY"],
    block: { min: 0, max: 100 }
  ) {
    pageInfo {
      hasNextPage
    }
    edges {
      node {
        id
        bundledIn {
          id
        }
        tags {
          name
          value
        }
      }
    }
  }
}
```


This does NOT work:
<img width="1248" alt="image" src="https://github.com/textury/arlocal/assets/84311757/26e44d22-25d1-4ef5-ba47-27b4e6cf182c">

Query:
```
query {
  transactions(
    ids: ["thLz97087UrlLfXJ0RphXZ7g_mK569ro63ulCxxRtdY"],
    block: { min: 0, max: null }
  ) {
    pageInfo {
      hasNextPage
    }
    edges {
      node {
        id
        bundledIn {
          id
        }
        tags {
          name
          value
        }
      }
    }
  }
}
```

The issue - latest versions of the ArLocal do not behave properly if `null` is passed as `max` in the block filter. I.e.
this filter works:
```
block: { min: 0, max: 100 }
```

This filter does NOT work:
```
block: { min: 0, max: null }
```

This apparently has changed in the latest ArLocal versions AND is inconsistent with how the arweave.net/graphql works:
<img width="1702" alt="image" src="https://github.com/textury/arlocal/assets/84311757/f291fb9f-44f8-4627-8a65-a949a6b3e840">

**Comment by erhant** [comment]
moving the discussion here (from https://github.com/warp-contracts/warp/issues/494#issuecomment-1878527671)

during this change https://github.com/textury/arlocal/compare/v1.1.62...v1.1.63 the removal of `|| undefined` causes `null` values to stay `null` instead of `undefined`, which the server probably does not handle.

**Comment by cedriking** [comment]
Merged and updated in version `v1.1.65`.
